tests/integration/features/features_int_test.go

```diff
@@ -373,7 +371,7 @@ var _ = Describe("feature cleanup", func() {
 				Expect(featuresHandler.Apply()).To(Succeed())
 
 				// then
-				service, err := getService("knative-local-gateway", namespace.Name)
+				service, err := fixtures.GetService("knative-local-gateway", namespace.Name, envTestClient)
```

Comment:
I would make it `client, ns, name` here.

Reply:
5686115